CMakeLists.txt

```diff
@@ -4,9 +4,12 @@ project(nlp VERSION 0.1.0)
 include(CTest)
 enable_testing()
 
-set(CMAKE_GENERATOR_PLATFORM Win32)
+find_package(ICU COMPONENTS uc i18n io REQUIRED)
```

Comment:
Try:  
```
find_package(ICU REQUIRED
  COMPONENTS uc i18n io
)
```

Reply:
Yea, I have seen both. I will give it a try.